### ISSUE TITLE
Make the camera button in the message screen working

### ISSUE THREAD

**Issue by cankush625** [cankush625/Instagram_Clone#8]
The camera feature is implemented in the code for the camera icon on the Instagram home screen. There is one more camera button on the message screen(the message button is in the top right corner), make this camera button working.

**Comment by mjmadhur** [comment]
Is it for hacktoberfest?

**Comment by cankush625** [comment]
Yes

**Comment by mjmadhur** [comment]
Can i solve this issue

**Comment by cankush625** [comment]
Sure. I will assign it to you. Fix it and make a PR

**Comment by mjmadhur** [comment]
I only need to solve this n is there any need to upload apk?

**Comment by cankush625** [comment]
No. You only have to raise PR with the solution for this issue.

**Comment by rajoriaakash** [comment]
Hi, Can I try this issue too for hacktoberfest?
